manager/apis/app/v1alpha1/blueprint_types.go

```diff
@@ -24,6 +24,10 @@ type AssetContext struct {
 	// +required
 	AssetID string `json:"assetID"`
 
+	// Capability of the module
+	// +required
+	Capability taxonomy.Capability `json:"capability"`
```

Comment:
Isn't better to enable more than one capability for the same asset? To have a list of Capability?

Reply:
I would assume the asset details for each capability would be different.